v2/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.8
+
+WORKDIR /chainwalkers_app
+
+COPY . .
+
+RUN pip3 install -r requirements.txt
+
+ENV PYTHONPATH "${PYTHONPATH}:/chainwalkers_app"
```

Comment:
maybe you can add in readme a description  how to use this docker image

Reply:
Added a new Readme for v2.